app/javascript/packs/webauthn-setup.js

```diff
@@ -1,19 +1,35 @@
-const WebAuthn = require('../app/webauthn');
+import { loadPolyfills } from '@18f/identity-polyfill';
+import { isWebAuthnEnabled, enrollWebauthnDevice } from '../app/webauthn';
+
+/**
+ * Reloads the current page, presenting the message corresponding to the given error key.
+ *
+ * @param {string} error Error key for which to show message.
+ */
+function reloadWithError(error) {
+  const params = new URLSearchParams(window.location.search);
+  params.set('error', error);
+  window.location.search = params.toString();
+}
 
 function webauthn() {
-  if (window.location.href.indexOf('?error=') === -1 && !WebAuthn.isWebAuthnEnabled()) {
-    window.location.search = '?error=NotSupportedError';
+  if (window.location.href.indexOf('?error=') === -1 && !isWebAuthnEnabled()) {
```

Comment:
Possible redirect loop here because `error` isn't guaranteed to be the first search parameter, so `indexOf` with `?` may not be present (it may be `&error=`).

Reply:
> Possible redirect loop here because `error` isn't guaranteed to be the first search parameter, so `indexOf` with `?` may not be present (it may be `&error=`).

Fixed in 4b6320b27.